amf-client/shared/src/test/resources/upanddown/somecars.raml

```diff
@@ -127,39 +127,6 @@ mediaType:
 version: 1.0.0
 protocols:
   - https
-/history:
```

Comment:
Ordening change. Lexical order should never have applied because we are passing from an OAS to a RAML